### PR TITLE
Merged online-docs into master

### PR DESCRIPTION
**Pull request by DavisAre**
Docs are ready, search added. Makes sense to merge into master.

### CODE REVIEW

File: .github/workflows/documentation.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+name: documentation
+
+on:
+  pull_request:
+    branches: [feature/master]
```
Comment (tharna) [inline .github/workflows/documentation.yml:5]: this should be just master to make things work after merging.

**Review by tharna — CHANGES_REQUESTED** [review]

File: .github/workflows/documentation.yml (hunk 1 above)
Comment (DavisAre) [reply .github/workflows/documentation.yml]: Sorry, my bad. Fixed.

**Review by tharna — APPROVED** [review]
Looks great!

**Comment by DavisAre** [comment]
FYI - @agnis-mateuss @Jancis: 
@tharna has reviewed the changes. Please consider a merge when possible.

**Review by agnis-mateuss — APPROVED** [review]

**Pull request merged**